packages/atomic/src/components/facets/base-facet/base-facet.tsx

```diff
@@ -56,30 +56,31 @@ export const BaseFacet: FunctionalComponent<BaseFacetProps> = (
     </button>
   ) : null;
 
+  const fullWidthEllipsedClasses =
```

Comment:
A generic question that I'd like to get your input on:

When we started atomic, I thought we would end up with more "internal" components, that would mostly share some tailwind styling with one another.

Right now, we pretty much have none of those.

If we were to do a inventory of all the UI we have in all components. and extract recurring theme (eg: a button should always look like this), do you think we would find that we have a ton of repetition or not ?

Do you think it would ultimately simplify or complexify the codebase to have those kind of components ?

Reply:
There isn't that much, but if we can identify them when we see the new design, we could come up with those new kind of components. Maybe functional components, but perhaps simple reusing common css classes with which we use `@apply` leveraging tailwind. https://tailwindcss.com/docs/functions-and-directives#apply

E.g, fullWidthEllipsedClasses, or facet label, could be a css class.